lib/bundler/cli/config.rb

```diff
@@ -109,7 +112,17 @@ def run
     #         the options conflict.
     #
     def resolve_system_path_conflicts(name, new_value, scope = "global")
-      if name == "path.system" and Bundler.settings[:path] and new_value == "true"
+      if name == "system" and Bundler.settings[:path] and new_value == "true"
+        Bundler.ui.warn "`path` is already configured, so it will be unset."
+        delete_config("path")
+        # We still want `system` to be set.
+        :no_conflict
+      elsif name == "path" and Bundler.settings[:system]
+        Bundler.ui.warn "`system` is already configured, so it will be unset."
+        delete_config("system")
+        # We still want `path` to be set.
+        :no_conflict
+      elsif name == "path.system" and Bundler.settings[:path] and new_value == "true"
```

Comment:
Oops. We wound up with `path`, `system`, and `path.system`. I intended us to only have two. :) I think `path` and `path.system` is the clearest. Let's get rid of just plain `system`.